portingdb/templates/howto.html

```diff
@@ -51,6 +51,14 @@ <h2 id="package-it">Package it</h2>
         </p><p>
             To let us know, edit the <i>Blocks</i> field of the Bugzilla report and add the string <b>PY3PATCH-AVAILABLE</b> (you can use space as a separator). That way it will appear in our <a href="https://bugzilla.redhat.com/show_bug.cgi?id=1312032">tracking bug</a> for packages with an available patch. If need be, we'll review your patch and push it for the maintainer.
         </p>
+        <h3>Fame and Glory</h3>
+        <p>
+            After you've done the deed, you deserve your reward! Sadly, the hand of the King's daughter and half of all the kingdom has recently become unavailable (may vary upon location, ask your nearest royal family), but fret not, you can get a <i>badge</i> of honour!
+        </p><p>
+            Visit the <a href="https://fedoraproject.org/wiki/Open_Badges/Python3Log">wiki page for badges</a> and add a row with your <i>FAS username</i> if it doesn't exist yet. Inside put a link to the Bugzilla report you helped resolve as well a link to the package on PortingDB. You can get 3 badges: for your 1st, 5th and also 10th resolved package!
```

Comment:
It might be good to tell people to wait a day or two – updating the mirrors and portingdb takes some time.